unit_test/test_eol.py

```diff
@@ -64,6 +68,22 @@ def assert_function_call(self, func, input_file_content, expected_output_file_co
             result = temp_dest.read().decode('utf-8')
             self.assertEqual(result, expected_output_file_content)
 
+    def assert_function_call_3args(self, func, input_file_content, expected_output_file_content, language):
```

Comment:
Let's rename `language` to `second_argument` to make the method generic

Reply:
Renamed.